inputs-required.tf

```diff
@@ -58,3 +58,22 @@ variable "tags" {
   type        = map(string)
   description = "The tags to apply to the virtual machine and associated resources."
 }
+
+variable "env" {
+  description = "The deployment environment (sandbox, aat, prod etc..)"
+}
+
+variable "cnp_vault_rg" {
+  type        = string
+  description = "The name of the resource group to deploy the virtual machine in."
+}
+
+variable "soc_vault_rg" {
+  type        = string
+  description = "The name of the vault to connect the virtual machine to."
+}
+
+variable "soc_vault_name" {
+  type        = string
+  description = "The name of the vault to connect the virtual machine to."
+}
```

Comment:
This will be a breaking change for anyone consuming this module - can we provide defaults for these and/or compute the values?